### PR TITLE
Utilize Nameof (part 0)

### PR DESCRIPTION
**Pull request by AdamSpeight2008**
Implement the usage of `nameof` rather that vanilla string literals.

### CODE REVIEW

**Comment by davkean** [comment]
tag @dotnet/roslyn-analysis 

Thanks for the PR, however, this mixes NameOf changes with formatting and functionality (Exts module) changes. You should remove the layout and functionality changes, these should be taken as separate commits and PRs.

File: src/Microsoft.VisualStudio.AppDesigner/Common/ShellUtil.vb [hunk 1]
```diff
@@ -68,12 +73,19 @@ Namespace Microsoft.VisualStudio.Editors.AppDesCommon
...
+$"Unable to get color from the shell, using a predetermined default color instead.
+Color Category = {themeCategory.ToString()}, Color Name = {themeColorName}, Color Type = {colorType}, Default Color = &h{ VB.Hex(defaultColor.ToArgb)}"
+)
             Return defaultColor
         End Function
 
         Private Shared Function RGBAToColor(ByVal rgbaValue As UInt32) As Color
-            Return Color.FromArgb(CInt((rgbaValue And &HFF000000UI) >> 24), CInt(rgbaValue And &HFFUI), CInt((rgbaValue And &HFF00UI) >> 8), CInt((rgbaValue And &HFF0000UI) >> 16))
+            Return Color.FromArgb(CInt((rgbaValue And &HFF000000UI) >> 24),
+                                  CInt(rgbaValue And &HFFUI),
```
Comment (davkean) [inline src/Microsoft.VisualStudio.AppDesigner/Common/ShellUtil.vb]: This an example of a formatting change that should be removed.

File: src/Microsoft.VisualStudio.AppDesigner/Common/Utils.vb [hunk 2]
```diff
@@ -138,18 +138,16 @@ Namespace Microsoft.VisualStudio.Editors.AppDesCommon
         ''' <remarks>Throws an internal exception if the bitmap cannot be found or loaded.</remarks>
         Public Function GetManifestImage(ByVal ImageID As String, Optional ByRef assembly As Assembly = Nothing) As Image
             Dim BitmapStream As Stream = GetType(Microsoft.VisualStudio.Editors.AppDesCommon.Utils).Assembly.GetManifestResourceStream(ImageID)
-            If Not assembly Is Nothing Then
```
Comment (davkean) [inline src/Microsoft.VisualStudio.AppDesigner/Common/Utils.vb]: This is another example of change that should be removed.

Comment (AdamSpeight2008) [inline src/Microsoft.VisualStudio.AppDesigner/Common/Utils.vb]: 'isNot; is more efficient

File: src/Microsoft.VisualStudio.AppDesigner/Common/ShellUtil.vb (hunk 1 above)
Comment (AdamSpeight2008) [inline src/Microsoft.VisualStudio.AppDesigner/Common/ShellUtil.vb]: I have an ultrawide monitor, use a size 10 font and I still require scrolling to right to see the signatures, That should be the case..

**Comment by davkean** [comment]
Adam, as previously mentioned, please remove the formatting and other changes unrelated to nameof - these can be submitted as a different PR. Feel free to open this PR when you've done that and we can look at it.

**Pull request closed without merging**